main/apcupsd/APKBUILD

```diff
@@ -1,28 +1,30 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=apcupsd
 pkgver=3.14.14
-pkgrel=0
+pkgrel=1
 pkgdesc="A Daemon to control APC UPSes"
-subpackages="$pkgname-doc $pkgname-webif"
+subpackages="$pkgname-doc $pkgname-webif $pkgname-openrc"
 url="http://www.apcupsd.org"
 arch="all"
 license="GPL-2.0"
```

Comment:
The license could be clarified : GPL-2.0-only - https://sourceforge.net/p/apcupsd/svn/HEAD/tree/branches/Branch-3_14/COPYING

Reply:
thanks, will fix